pkg/apis/api.acorn.io/v1/computeclass.go

```diff
@@ -0,0 +1,28 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ComputeClass struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	Spec MinComputeClassSpec `json:"spec,omitempty"`
```

Comment:
There doesn't need to be a spec field.  It's not often clear on whether to use spec/status or not but the general rule of thumb is that objects that are orchestrated and have orchestration status should be spec/status.  For this object it's purely a data object, more like a secret or configmap.  So it's fields can just be top level.  So

```go
type ComputeClass struct {
  Descriptiong string
  }
  ```